packages/documentation/src/stories/components/textarea/textarea.stories.tsx

```diff
@@ -0,0 +1,300 @@
+      control: {
+        type: 'text'
+      },
+      table: {
+        category: 'States'
+      }
+    },
+    invalidFeedback: {
+      name: 'Invalid Feedback',
+      description: 'Text to show when the component is in an invalid state.',
+      control: {
+        type: 'text'
+      },
+      table: {
+        category: 'States'
+      }
+    }
+  }
+} as Meta;
+
+const Template = (args: Args, story: Story) => {
+  const id = `ExampleTextarea_${story.name}`;
+  const classes = [
+    'form-control',
+    args.size,
+    args.validation
+  ].filter(c => c && c !== 'null').join(' ');
+
+  const component = <textarea
+    id={ id }
+    className={ classes }
+    placeholder=" "
+    rows={ args.rows }
+    disabled={ args.disabled }
+    aria-label={ args.label }
+    aria-invalid={ VALIDATION_STATE_MAP[args.validation] }
+  ></textarea>;
+
+  const contextuals: (JSX.Element | null)[] = [
+    args.validation === 'is-valid' ? <p key="valid" className="valid-feedback">{ args.validFeedback }</p> : null,
+    args.validation === 'is-invalid' ? <p key="invalid" className="invalid-feedback">{ args.invalidFeedback }</p> : null,
+    args.hint !== '' ? <div key="hint" className="form-text">{ args.hint }</div> : null
+  ].filter(f => f !== null);
+
+  if (args.floatingLabel) {
+    return <div className="form-floating">
+      { component }
+      <label htmlFor={ id } className="form-label">{ args.label }</label>
+      { contextuals }
+    </div>;
+  } else { 
+    return [component, contextuals];
+  }
+};
+
+export const Default = Template.bind({}) as Meta;
+
+export const FloatingLabel = Template.bind({}) as Meta;
+FloatingLabel.parameters = {
+  controls: {
+    exclude: [
+      'Size',
+      'Rows',
+      'Helper Text',
+      'Disabled',
+      'Validation',
+      'Valid Feedback',
+      'Invalid Feedback'
+    ]
+  }
+};
+FloatingLabel.args = {
+  floatingLabel: true,
+  hint: ''
+};
+
+export const Size = Template.bind({}) as Meta;
+Size.parameters = {
+  controls: {
+    exclude: [
+      'Label',
+      'Floating Label',
+      'Rows',
+      'Helper Text',
+      'Disabled',
+      'Validation',
+      'Valid Feedback',
+      'Invalid Feedback'
+    ]
+  }
+};
+Size.args = {
+  size: 'form-control-sm',
+  hint: ''
+};
+
+export const Rows = Template.bind({}) as Meta;
+Rows.parameters = {
+  controls: {
+    exclude: [
+      'Label',
+      'Floating Label',
+      'Size',
+      'Helper Text',
+      'Disabled',
+      'Validation',
+      'Valid Feedback',
+      'Invalid Feedback'
+    ]
+  }
+};
+Rows.args = {
+  rows: 8,
+  hint: ''
+};
+
+export const HelperText = Template.bind({}) as Meta;
+HelperText.parameters = {
+  controls: {
+    exclude: [
+      'Label',
+      'Floating Label',
+      'Size',
+      'Rows',
+      'Disabled',
+      'Validation',
+      'Valid Feedback',
+      'Invalid Feedback'
+    ]
+  }
+};
+
+export const Disabled = Template.bind({}) as Meta;
```

Comment:
I'm not sure this story adds a lot of value. There is nothing special to keep in mind and no addidtional docs. In my opinion, the default story captures this use-case.

Reply:
Resolving this because of the accessibility comment https://github.com/swisspost/design-system/pull/833#discussion_r1038003184